reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -321,278 +310,308 @@ class StaticHtmlReporter : Reporter() {
-                        <tr>
-                            <th>Source</th>
-                            <th>Error</th>
-                        </tr>
-                        </thead>
-                        <tbody>
-                        """.trimIndent())
-
-                    evaluatorErrors.forEach { error ->
-                        val cssClass = when {
-                            error.isResolved -> "ort-resolved"
-                            else -> "ort-error"
+                        projectTable.exclude?.let { exclude ->
+                            +" "
+                            div("ort-reason") { +"Excluded: ${exclude.reason} - ${exclude.comment}" }
                         }
-                        append("""
-                            <tr class="$cssClass">
-                                <td>${error.source}</td>
-                                <td><p>${error.description}</p><p>${error.resolutionDescription}</p></td>
-                            </tr>
-                            """.trimIndent())
                     }
-
-                    append("""
-                        </tbody>
-                        </table>
-                        """.trimIndent())
                 }
             }
+        }
+    }
 
-    private fun createErrorTable(title: String, errors: ErrorTable, anchor: String) =
-            buildString {
-                append("""
-                    <h2 id="$anchor">$title</h2>
-                    <p>Errors from excluded components are not shown in this summary.</p>
-                    <h3>Packages</h3>
-                    <table class="ort-report-table ort-errors">
-                    <thead>
-                    <tr>
-                        <th>Package</th>
-                        <th>Analyzer Errors</th>
-                        <th>Scanner Errors</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    """.trimIndent())
-
-                errors.rows.forEach { row ->
-                    val cssClass = "ort-error"
-
-                    append("""
-                        <tr class="$cssClass">
-                            <td>${row.id}</td>
-                            <td>""".trimIndent())
-
-                    row.analyzerErrors.forEach { id, errors ->
-                        append("""
-                                <a href="#$id">$id</a>
-                                <ul>
-                                ${errors.joinToString("\n") {
-                                    "    <li><p>${it.description.replace("\n", "</br>")}</p>" +
-                                            "<p>${it.resolutionDescription}</p></li>"
-                                }}
-                                </ul>""".trimIndent())
+    private fun DIV.evaluatorTable(evaluatorErrors: List<ResolvableIssue>) {
+        h2 {
+            id = "policy-violation-summary"
+            +"Rule Violation Summary (${evaluatorErrors.size} violations)"
```

Comment:
Hmm, here on the other hand we have "violations" again. I think we should consistently either keep it or drop it, and do the same for errors.

Reply:
I decided to add "violations" and "errors", just to make sure people understand what that number means.